services/galley/src/Galley/Intra/Client.hs

```diff
@@ -40,3 +51,51 @@ notifyClientsAboutLegalHoldRequest requesterUid targetUid lastPrekey' prekeys =
                 . path "/i/clients/legalhold/request"
                 . json (LegalHoldClientRequest requesterUid targetUid lastPrekey' prekeys)
                 . expect2xx
+
+getLegalHoldAuthToken :: UserId -> Galley OpaqueAuthToken
+getLegalHoldAuthToken uid = do
+    (brigHost, brigPort) <- brigReq
+    r <- call "brig" $
+           method POST
+            . host brigHost
+            . port brigPort
+            . path "/i/sso-login" -- ^ TODO: switch to '/i/legalhold-login'
+            . queryItem "persist" "true"
+            . json (SsoLogin uid Nothing)
+            . collectCookies
+            . expect2xx
+    lg <- view applog
+    case getCookieValue "zuid" r of
```

Comment:
I tried setting a custom cookie label but it didn't work; maybe that's something other than what I thought? At any rate, using `zuid` works.

This will all change when we switch to the new token endpoint so don't get too hung up on it 😋

Reply:
:+1: yes, it has to be zuid, or whatever name we have once https://github.com/wireapp/wire-server/pull/761 is ready.

Reply:
Do you happen to know what the `CookieLabel` parameter on the request to `sso-login` is for? I must be misunderstanding its purpose then.

Reply:
after off-line discussion: may be documented in swagger.  it's used by clients to reference individual cookies they want to e.g. delete.